compiler/cpp/src/thrift/generate/t_py_generator.cc

```diff
@@ -127,6 +128,9 @@ class t_py_generator : public t_generator {
         gen_tornado_ = true;
       } else if( iter->first.compare("coding") == 0) {
         coding_ = iter->second;
+      } else if( iter->first.compare("type_hints") == 0) {
+        gen_type_hints_ = true;
+        gen_enum_ = true;
```

Comment:
should the help text below mention that the enum option is also enabled if this option is enabled?

Reply:
added it, but it should be obvious as without enum type type hinting won't work.